src/VisualStudio/IntegrationTest/IntegrationTests/Extensions/CommonExtensions.cs

```diff
@@ -35,6 +35,13 @@ public static void InvokeCodeActionList(this AbstractIntegrationTest test)
             test.WaitForAsyncOperations(FeatureAttribute.LightBulb);
         }
 
+        public static void InvokeCodeActionListWithoutWaiting(this AbstractIntegrationTest test)
+        {
+            // compare to InvokeCodeActionList() which waits for various async waiters
```

Comment:
Doc comment with cref? (and maybe a doc comment on the other one pointing to this one as well, so callers of either can see what they're doing without GTD?)